test/integration/apiserver/apiserver_test.go

```diff
@@ -1677,6 +1678,193 @@ func TestGetSubresourcesAsTables(t *testing.T) {
 	}
 }
 
+func TestGetScaleSubresourceAsTableForAllBuiltins(t *testing.T) {
```

Comment:
This uses `test/integration/framework/test_server.go` instead of `cmd/kube-apiserver/app/testing/testserver.go` so it is missing the `api/all=true` runtime config.

You also want the following to really make sure all APIs are enabled:

```go
	t.Setenv("KUBE_APISERVER_SERVE_REMOVED_APIS_FOR_ONE_RELEASE", "true")

	featuregatetesting.SetFeatureGateDuringTest(t, feature.DefaultFeatureGate, "AllAlpha", true)
	featuregatetesting.SetFeatureGateDuringTest(t, feature.DefaultFeatureGate, "AllBeta", true)
```

Reply:
Pushed an update.